## 公司集群使用过程中遇到的问题
1. digital_warehouse之前跑的时候,去Hadoop webui界面看这个任务执行情况时,发现任务在很多节点都被杀了,根据输出日志发现那些节点收到个终止信号,还有多个jar包的版本不对

今天跑这个表出现Caused by: org.codehaus.janino.JaninoRuntimeException: Code of method "evalExpr3$(Lorg/apache/spark/sql/catalyst/expressions/GeneratedClass$SpecificUnsafeProjection;Lorg/apache/spark/sql/catalyst/InternalRow;)V" of class "org.apache.spark.sql.catalyst.expressions.GeneratedClass$SpecificUnsafeProjection" grows beyond 64 KB
还打印了几万行不知道什么的日志
2. 205上用yarn调度时小表能跑,但是跑大表就会报parquet流的id不匹配.
3. 模型在207上面跑时写的表,用spark SQL读不出来,但是用hive可以读出来;模型在205上面跑时写的数据,没有这个问题.
4. 跑数据时发现经常有些节点上面的容器经常失败,输出日志大致如下
WARN scheduler.TaskSetManager: Lost task 116.0 in stage 2.0 (TID 238, data2, executor 2): ExecutorLostFailure (executor 2 exited caused by one of the running tasks) Reason: Container marked as failed: container_1547466357861_0549_01_000004 on host: data2. Exit status: 1. Diagnostics: Exception from container-launch.
Container id: container_1547466357861_0549_01_000004
